app/assets/stylesheets/components/_util.scss

```diff
@@ -1,5 +1,9 @@
 // supplemental utility classes
 
+// scss-lint:disable ImportantRule
+.display-block { display: block !important; }
```

Comment:
I'd assume bass css has a class for `display-block` -- perhaps if we want to amend that to be `important` we can name it something like `display-block-important`? (up to the CSSexperts of course)